### PR TITLE
add note permission,logout so that people cannot access my notes 

### PR DESCRIPTION
**Pull request by wind0727**
### What is this PR for?
resolve  logout so that people cannot access my notes, Notebook to modify the permissions will pop up, and provide the login button, rather than the login button on the home page

### What type of PR is it?
[ Improvement ]

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-723

### How should this be tested?
1. create a notebook
2. click Note Permission
3. Enter comma separated users and groups,ex: owners:admin,user1; readers:*;writers:admin,user1
4. click "Run this paragraph", warn "only admin,user1 can write this notebook ", prompt box show "Login" button
5. click "Login" button, userName:user1,password:password2, after login success, click "Run this paragraph", can access this notebook
6. click "Logout" button,cannot access this notebook

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?

### CODE REVIEW

**Comment by wind0727** [comment]
Failed to execute goal com.github.eirslett:frontend-maven-plugin:0.0.25:grunt (grunt build) on project zeppelin-web: Failed to run task: 'grunt build --no-color' failed. (error code 3) -> [Help 1]

**Comment by Leemoonsoo** [comment]
Thanks for the contribution. 
Last CI test shows error
```
[INFO] src/components/navbar/navbar.controller.js
[INFO]   line 77  col 33  Strings must use singlequote.
[INFO]   line 78  col 41  Strings must use singlequote.
[INFO]   line 79  col 38  Strings must use singlequote.
[INFO]   line 80  col 37  Strings must use singlequote.
[INFO] 
[INFO] src/components/websocketEvents/websocketEvents.factory.js
[INFO]   line 71  col 17  '$' is not defined.
[INFO] 
[INFO] ✖ 5 problems
[INFO] 
[INFO] Warning: Task "jshint:all" failed. Use --force to continue.
[INFO] 
[INFO] Aborted due to warnings.
```
@wind0727 could you take care of those problems?

**Comment by wind0727** [comment]
'$' How to define?

**Comment by Leemoonsoo** [comment]
'angular.element' can be used instead of '$'

**Comment by wind0727** [comment]
OK,Thanks!

File: zeppelin-web/src/components/navbar/navbar.html [hunk 1]
```diff
@@ -83,7 +81,10 @@
         <li ng-if="!ticket">
             <button class="btn btn-default" data-toggle="modal" data-target="#loginModal" ng-click="showLoginWindow()" style="margin-left: 10px">Login</button>
         </li>
+        <li ng-show="userName != ''">
```
Comment (prabhjyotsingh) [inline zeppelin-web/src/components/navbar/navbar.html]: this should be `ticket.principal != 'anonymous' `

File: zeppelin-web/src/components/navbar/navbar.html [hunk 2]
```diff
@@ -83,7 +81,10 @@
...
+        	<button type="button" class="btn btn-default btn-xs" ng-click="logout()" tooltip-placement="bottom" tooltip="logout">{userName}}</button>
```
Comment (prabhjyotsingh) [inline zeppelin-web/src/components/navbar/navbar.html]: any reason why you need to say `{userName}}` instead of `Logout`

Comment (wind0727) [inline zeppelin-web/src/components/navbar/navbar.html]: Thanks for reminding,I will modify this right now.

**Comment by anthonycorbacho** [comment]
changes look good to me

**Comment by wind0727** [comment]
It's my pleasure.

**Comment by prabhjyotsingh** [comment]
LGTM

**Comment by prabhjyotsingh** [comment]
Merge is no more discussion.

**Pull request closed without merging**